docs/decisions/0002-forum_v1_migration_to_v2.rst

```diff
@@ -54,6 +54,148 @@ Migration Decisions
 * **Functionality**: This method was used in the ``threads`` and ``comments`` APIs to filter the content of a thread or comment by checking against a set of ``blocked_hashes`` stored in MongoDB. If the content matched a blocked hash, the API would return a 503 error.
 * **Reason**: There was no mechanism in place in the current discussion forum to add or maintain these blocked hashes. As a result, the method became irrelevant and was not included in the new system.
 
+6. Improvement in elasticsearch indexes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Issues in V1**:
+    - In version 1, the entire data from MongoDB was imported into Elasticsearch, which resulted in larger documents that did not align with the mappings defined in Elasticsearch.
```

Comment:
may be we can explain it more, increase the length using points:
* **Issues in V1**
- In the initial implementation (v1), the entire dataset from MongoDB was imported directly into Elasticsearch.
- This led to larger document sizes in Elasticsearch, many of which contained unnecessary fields that did not align with the pre-defined mappings in Elasticsearch.
- As a result, performance was negatively impacted, with higher storage costs and slower query responses.

* **Optimization in v2**:
In version 2, significant improvements were made to enhance the indexing process:

- Only the fields defined in Elasticsearch mappings were migrated from MongoDB.
- This reduced the size of each document, improving performance and reducing storage overhead.
- The optimized mappings streamlined query performance and index management.